addon/components/lt-body.js

```diff
@@ -239,6 +292,57 @@ export default Component.extend({
     this.set('useVirtualScrollbar', fixedHeader || fixedFooter);
   },
 
+  didReceiveAttrs() {
+    const scrollTo = this.get('scrollTo');
+    const _scrollTo = this.get('_scrollTo');
+    const showRow = this.get('showRow');
+    const _showRow = this.get('_showRow');
+
+    this.set('_scrollTo', scrollTo);
+    this.set('_showRow', showRow);
+
+    if (scrollTo !== _scrollTo) {
+      this.set('targetScrollOffset', scrollTo);
+      this.set('hasReachedTargetScrollOffset', false);
+      run.scheduleOnce('afterRender', this, this.checkTargetScrollOffset);
```

Comment:
Also, none of this should be happening if `scrollTo` and `scrollToRow` are not set.